pronto_ros/src/ins_ros_handler.cpp

```diff
@@ -16,19 +16,18 @@ InsHandlerROS::InsHandlerROS(ros::NodeHandle &nh) : nh_(nh)
     std::string imu_frame = "imu";
 
     nh_.getParam(ins_param_prefix + "frame", imu_frame);
-    const std::string base_frame = "base";
-    Eigen::Affine3d ins_to_body;
-    while(nh_.ok()){
-        try{
-            geometry_msgs::TransformStamped temp_transform;
-            temp_transform = tf_imu_to_body_buffer_.lookupTransform(base_frame, imu_frame, ros::Time(0));
-
+    std::string base_frame = "base";
```

Comment:
better print what `base_frame` is going to be. Also maybe `base_frame_id` is a better name? I'm ok with both.